frigate/embeddings/embeddings.py

```diff
@@ -266,10 +268,7 @@ def reindex(self) -> None:
         # Get total count of events to process
         total_events = (
             Event.select()
-            .where(
-                (Event.has_clip == True | Event.has_snapshot == True)
-                & Event.thumbnail.is_null(False)
```

Comment:
Won't removing this prevent old thumbnail-only events from being indexed?

Reply:
Why would it? This check was never needed because thumbnail is not null was previously guaranteed due to the schema. 

And now we can't have that check otherwise we won't be able to reindex events that have thumbnails on disk

Reply:
Right. Just making sure I understand, if someone has a bunch of old events in their db that only has a thumbnail and no clip or snapshot, those won't be indexed anymore, right?

Reply:
Ah I see what you mean, I'll adjust this